docs/install/README.md

```diff
@@ -38,21 +38,36 @@ By default kind clusters run on the same Docker network which means we will have
 # Single Cluster Install with Kustomize
 We will first look at a single cluster install to demonstrate that while K8ssandra Operator is designed for multi-clluster use, it can be used in a single cluster without any extran configuration.
 
-## Create kind cluster
+## Automated Setup
+Run `make kind-setup` to create a single kind cluster and deploy k8ssandra-operator along with its dependencies.  
+Check that there are two Deployments. The output should look similar to this:
+
+```
+kubectl get deployment
+NAME                 READY   UP-TO-DATE   AVAILABLE   AGE
+cass-operator        1/1     1            1           2m
+k8ssandra-operator   1/1     1            1           2m
+```
+
+The operator will be deployed in the `default` namespace using this procedure.
+
+## Manual Setup
```

Comment:
Confirming single cluster manual setup works as expected. 

Minor note - the `heredoc` for the `K8ssandraCluster` resource has excess whitespace after the terminating `EOF` which will cause it not to be evaluated as a `heredoc`. Be good to remove this for those unfamiliar with this piece of shell magic.

Reply:
Testing multicluster manual installation:

Running the recommended `create-clientconfig.sh` command as below 

```
./create-clientconfig.sh --src-kubeconfig build/kubeconfigs/k8ssandra-1.yaml --dest-kubeconfig build/kubeconfigs/k8ssandra-0.yaml --in-cluster-kubeconfig build/kubeconfigs/updated/k8ssandra-1.yaml --output-dir clientconfig
```

This yields an error: `At least one of the --src-context or --src-kubeconfig options must be specified`

Reply:
> Minor note - the heredoc for the K8ssandraCluster resource has excess whitespace after the terminating EOF which will cause it not to be evaluated as a heredoc. Be good to remove this for those unfamiliar with this piece of shell magic.

Good catch, I'll remove the extra whitespace.